src/UserList.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import {useState} from 'react'
+import { useEffect } from 'react'
```

Comment:
If you 2 or more named imports from the same package, you don't need to make it in multiple lines, you can make it like this:

```js
import { useState, useEffect } from 'react';
```